### PR TITLE
Fixed the trait schema changes and arranged the samples in folders

### PR DESCRIPTION
**Pull request by sowsan**
- Fixed the trait schema changes and arranged the samples in folders
- Added the Helloworld example from the rudr repo
- Added a generic readme file in the root

Thanks to @wdfox for testing and verifying these new changes!

### CODE REVIEW

**Comment by hongchaodeng** [comment]
fix https://github.com/oam-dev/rudr/issues/519

Please rebase :)

**Comment by sowsan** [comment]
haven't done that before :), you mean just "git rebase origin/master"

**Comment by hongchaodeng** [comment]
Well.. Not _origin/master_ actually. It should be upstream.
I am not that good at explaining git, and it is hard to explain...
Could you ask around your colleagues who know about git?

**Comment by sowsan** [comment]
> Well.. Not _origin/master_ actually. It should be upstream.
> I am not that good at explaining git, and it is hard to explain...
> Could you ask around your colleagues who know about git?

sure will do.

**Comment by ryanzhang-oss** [comment]
@sowsan  https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork
hope this helps. 
We are looking forward to merge your PR

File: 2.ServiceTracker_App/App/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+In this example, there are various roles that handle each aspect of the OAM application.
+
+* UI Developer
+* API Microservices Developer
+* MongoDB Admin
+* App Operator / SRE (handles application deployment in Kubernetes)
+
+## Quickstart: Deploy the sample to Rudr running on any Kubernetes cluster
+
```
Comment (ryanzhang-oss) [inline 2.ServiceTracker_App/App/README.md]: What's the difference between this README and the one in ServiceTracker_App/README.md?

Comment (sowsan) [reply 2.ServiceTracker_App/App/README.md]: Thanks for finding it. It was duplicate and removed it.

**Comment by sowsan** [comment]
> @sowsan https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork
> hope this helps.
> We are looking forward to merging your PR

Thank you. Added 

image

**Comment by ryanzhang-oss** [comment]
@sowsan  Thank you very much for working on this. Can you please resolve the conflicts below?

app/data-api/package-lock.json
app/data-api/package.json
tracker-app-config.yaml

**Comment by sowsan** [comment]
Sure I will work on and push it in next couple of days

**Comment by sowsan** [comment]
@hongchaodeng @ryanzhang-oss  , sorry for the long delay. I just finished fixing of conflicts.

**Comment by hongchaodeng** [comment]
Thanks @sowsan 

lgtm

**Pull request merged**